src/main/java/nl/tudelft/planningstool/database/entities/courses/CourseRelation.java

```diff
@@ -0,0 +1,39 @@
+package nl.tudelft.planningstool.database.entities.courses;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import nl.tudelft.planningstool.database.entities.User;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Data
+@Entity
+@Table(name = "course_relations")
+@EqualsAndHashCode(of = {
+        "course", "user"
+})
+public class CourseRelation implements Serializable {
```

Comment:
> for hours

The dedication is real! Thanks for figuring this one out. That website tho... welcome in 1995